docs/content/architecture/disaster-recovery.md

```diff
@@ -210,6 +210,9 @@ Cluster:
   BackrestS3Bucket: my-postgresql-backups-example
   BackrestS3Endpoint: s3.amazonaws.com
   BackrestS3Region: us-east-1
+  BackrestS3URIStyle: host
+  BackrestS3VerifyTLS: true
```

Comment:
I believe we're default this to `false`, so it may make sense to default that here as well.

Reply:
Ignore, I don't know where this came from.